### DISCUSSION TITLE
Firebase Database Support

### DISCUSSION THREAD

**Discussion by imuhammadarsalan** [directus/directus#2825] [Feature Requests]
Hi,

Your CMS is awesome but it doesn't fit in our requirement. We want to use firebase for realtime communication since we looking our self how we can add support firebase in this project any suggestion will be appreciated from dev team.

**Comment by benhaynes** [comment]
We've had a decent number of requests for this, but unfortunately our platform is built to support SQL, and Firebase is NoSQL. We'd love to add support for Mongo and Firebase in the future, but it certainly won't be happening in the short-term.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
